### PR TITLE
Add pendulum

### PR DESCRIPTION
**Pull request by JulienCalbert**
- added a code to generate a urdf file for the double pendulum
- added a folder for file management in Utils
- simplify (speed up) the test for the PWA system and the cover based on the ellispoids
- replace the pyplot module in the doc by Plots except for [Example: Gol, Lazar and Belta (2013)](https://dionysos-dev.github.io/Dionysos.jl/dev/generated/Gol,%20Lazar%20&%20Belta%20(2013)/#Example:-Gol,-Lazar-and-Belta-(2013))
- modify the SCOTS solver to make it clearer (i.e. separate the construction of the abstraction and the solution of the control problem)

### CODE REVIEW

File: examples/example_SF_abst_2-simple.jl [hunk 1]
```diff
@@ -0,0 +1,250 @@
...
+#building abstraction
+@time Dionysos.Symbolic.compute_symmodel_from_hybridcontrolsystem!(symmodel, transitionCost, transitionKappa, system, W, L, U, opt_sdp, opt_qp)
+
+# Define Specifications
+x0 = SVector{n_sys}(problem.initial_set) # initial condition
+Xinit = Dionysos.Domain.DomainList(Xgrid)
+Dionysos.Domain.add_coord!(Xinit, problem.initial_set)
+Xfinal = Dionysos.Domain.DomainList(Xgrid) # goal set
+Dionysos.Domain.add_coord!(Xfinal, SVector{n_sys}(problem.target_set))
+#Dionysos.Domain.add_set!(Xfinal,Dionysos.Domain.HyperRectangle(SVector(-2.0, 0.5), SVector(-2.0, 0.5)), Dionysos.Domain.OUTER)
```
Comment (adrienbanse) [inline examples/example_SF_abst_2-simple.jl:84]: Remove ?

File: examples/example_SF_abst_2-simple.jl [hunk 2]
```diff
@@ -0,0 +1,250 @@
...
+end
+
+initlist = [Dionysos.Symbolic.get_state_by_xpos(symmodel, pos) for pos in Dionysos.Domain.enum_pos(Xinit)]; 
+finallist = [Dionysos.Symbolic.get_state_by_xpos(symmodel, pos) for pos in Dionysos.Domain.enum_pos(Xfinal)];
+filterTab = [pos for pos in Dionysos.Domain.enum_pos(Xobstacles)]
+filtered_array = filter(x->Dionysos.Symbolic.is_in(symmodel, x), filterTab)
+obstaclelist = [Dionysos.Symbolic.get_state_by_xpos(symmodel, pos) for pos in filtered_array]; 
+# Synthesis of the discrete controler
+
+contr = Dionysos.Control.NewControllerList();
```
Comment (adrienbanse) [inline examples/example_SF_abst_2-simple.jl:99]: We could define `const CO = Dionysos.Control` for homogeneity

Comment (adrienbanse) [reply examples/example_SF_abst_2-simple.jl:99]: Same for symbolic, etc.

File: problems/double_pendulum/double_pendulum_symbolic.jl [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+const PB = DI.Problem
+const ST = DI.System
+const CO = DI.Control
+const SY = DI.Symbolic
+
+Random.seed!(123)
+
+include("double_pendulum_urdf.jl")
+
+# les moments d'inerties ne semblent pas etre les memes ?  q2(eux) = q2(moi)-q1
```
Comment (adrienbanse) [inline problems/double_pendulum/double_pendulum_symbolic.jl:17]: It should be written in english, or be removed

**Review by adrienbanse — COMMENTED** [review]
Thanks @JulienCalbert 🥳. We should write a test for the double pendulum, maybe open an issue about that?

File: problems/double_pendulum/double_pendulum_symbolic.jl (hunk 3 above)
Comment (JulienCalbert) [reply problems/double_pendulum/double_pendulum_symbolic.jl:17]: I removed

**Pull request merged**